src/callbacks.js

```diff
@@ -78,13 +81,21 @@ jQuery.Callbacks = function( options ) {
 				}
 			}
 			firing = false;
+
+			// If not disabled,
```

Comment:
I suppose not, but it will be gone anyway if we land the followup: https://github.com/gibson042/jquery/compare/2015-01-callbacks...2015-01-callbacks-reduce#diff-67fa1bedebcff27953960d35db47f26bL85